docs/buildah-add.md

```diff
@@ -65,32 +65,32 @@ end with .go that are found in all directories.
 Example .dockerignore file:
 
 ```
-# comment
-*/temp*
-**/temp*
-temp?
+# here are files we want to exclude
+*/*.c
+**/output*
+src
 ```
 
-`*/temp*`
-Excludes files and directories whose names start with temp in any immediate subdirectory of the root. For example, the plain file /somedir/temporary.txt is excluded, as is the directory /somedir/temp.
+`*/*.c`
+Excludes files and directories whose names ends with .c in any top level subdirectory. For example, the source file include/rootless.c.
 
-`**/temp*`
-Excludes files and directories starting with temp from any subdirectory.
+`**/output*`
+Excludes files and directories starting with `output` from any directory.
 
-`temp?`
-Excludes files and directories in the root directory whose names are a one-character extension of temp. For example, /tempa and /tempb are excluded.
+`src`
+Excludes files named src and the directorie src as well as all content in it.
```

Comment:
the any/all bit is a suggestion, your call, directorie needs to be changed though
```suggestion
Excludes files named src and the directory src as well as any content in it.
```